src/Components/QuestionList/QuestionFilter/QuestionFilters.tsx

```diff
@@ -28,7 +28,7 @@ export const QuestionFilters = ({ filterSettings }: Props) => {
                                     type: typeSelect,
                                     name: nameSearch,
                                     hideCorrect: hideCorrect,
-                                    difficulty: difficulty,
+                                    difficulty: difficulty as Difficulty,
```

Comment:
If you did your typing correctly, you shouldn't need the `as Difficulty`. That's only needed when you are doing something like `Difficulty` => `any` => `Difficulty`, which you shouldn't be doing